traits/etsconfig/tests/test_etsconfig.py

```diff
@@ -10,10 +10,7 @@
 import tempfile
 import time
 
-if sys.version_info[:2] == (2, 6):
-    import unittest2 as unittest
-else:
-    import unittest
+import unittest
```

Comment:
Thanks. I should have caught this one as part of #446. :-(  Just double-checked; I don't see any other uses of `unittest2` in the codebase.